samples/index.md

```diff
@@ -44,7 +23,43 @@ Run popular software using Docker.
+| [global-2018-hol](https://github.com/dockersamples/global-2018-hol){: target="_blank" rel="noopener" class="_"} | Labs about modernizing traditional .NET / Windows applications. Included are a number of other labs on a variety of topics and for different skill levels, including beginners. |
+| [gopher-task-system](https://github.com/dockersamples/gopher-task-system){: target="_blank" rel="noopener" class="_"} | A simple task system using Go Docker SDK. |
+| [javaee-demo](https://github.com/dockersamples/javaee-demo){: target="_blank" rel="noopener" class="_"} | An example that demonstrates the path to modernizing a Java EE application to a containerized infrastructure. |
+| [k8s-wordsmith-demo](https://github.com/dockersamples/k8s-wordsmith-demo){: target="_blank" rel="noopener" class="_"} | Wordsmith demo project that runs across three containers: a Postgres database, a Java REST API, and a Go web application. |
+| [link-shortener-typescript](https://github.com/dockersamples/link-shortener-typescript){: target="_blank" rel="noopener" class="_"} | A sample URL shortener using TypeScript, Nest and Docker Desktop. |
+| [linux_tweet_app](https://github.com/dockersamples/linux_tweet_app){: target="_blank" rel="noopener" class="_"} | A sample NGINX website that allows a user to send a tweet. |
+| [newsletter-signup](https://github.com/dockersamples/newsletter-signup){: target="_blank" rel="noopener" class="_"} | A sample .NET Framework app using Docker containers on Windows. |
+| [node-bulletin-board](https://github.com/dockersamples/node-bulletin-board){: target="_blank" rel="noopener" class="_"} | A sample buletin board application using Vue.js.  |
+| [retail-store-items-detection-nodered](https://github.com/dockersamples/retail-store-items-detection-nodered){: target="_blank" rel="noopener" class="_"} | Build and deploy a retail store items detection system using Node-RED on an IoT Edge device.  |
+| [single-dev-env](https://github.com/dockersamples/single-dev-env){: target="_blank" rel="noopener" class="_"} | A single container sample of Docker Dev Environments. |
+| [slack-clone-docker](https://github.com/dockersamples/https://github.com/dockersamples/slack-clone-docker){: target="_blank" rel="noopener" class="_"} | Containerize a Slack clone app using the MERN stack. |
+| [spring-petclinic-docker](https://github.com/dockersamples/spring-petclinic-docker){: target="_blank" rel="noopener" class="_"} | A sample Spring Boot application built using Maven. |
+| [student-record-management](https://github.com/dockersamples/student-record-management){: target="_blank" rel="noopener" class="_"} | Containerize a .NET application using Docker Compose. |
+| [windows-2003](https://github.com/dockersamples/windows-2003){: target="_blank" rel="noopener" class="_"} | Three examples of moving sample apps from Windows 2003 to containers that can be ran on Windows Server 2016 in a data center or in any cloud.|
```

Comment:
This one looks very old as well (last updated 4 years ago)

Are we confident all example in this list are maintained and representable (otherwise it may be better _not_ to link them)

Reply:
Thanks @thaJeztah. The older samples have been removed from the table. Now, only a few popular samples are shown to give the user an idea about what's contained in the repos.